rulerr.gemspec

```diff
@@ -6,8 +6,8 @@ require 'rulerr/version'
 Gem::Specification.new do |spec|
   spec.name          = "rulerr"
   spec.version       = Rulerr::VERSION
-  spec.authors       = ["Ricardo Bernardeli"]
-  spec.email         = ["ricardobcs@gmail.com"]
+  spec.authors       = ["Ricardo Bernardeli", "Erik Ecoologic"]
+  spec.email         = %w(ricardobcs@gmail.com erikecoologic@gmail.com)
```

Comment:
The `email` attribute is optional, you can completly remove this line if you don't want to leave your email public almost asking for spam :p